cloud/aws/client/fake/fake.go

```diff
@@ -104,3 +104,16 @@ func (c *awsClient) RegisterInstancesWithLoadBalancer(input *elb.RegisterInstanc
 func NewClient(kubeClient kubernetes.Interface, secretName, namespace, region string) (client.Client, error) {
 	return &awsClient{}, nil
 }
+
+// GetRunningInstances returns instances that match InstanceStateNameRunning or InstanceStateNamePending
+func (c *awsClient) GetRunningInstances(clusterID string) ([]*ec2.Instance, error) {
```

Comment:
https://github.com/openshift/cluster-api-provider-aws/pull/57 will most likely ets merged before this one so it will require to regen and rebase. Though, the changes are minimal.

Reply:
Thanks for hinting on that.